src/test/java/study/OperatorTest.java

```diff
@@ -0,0 +1,36 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import util.Operator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OperatorTest {
+    private Operator operator;
+    @DisplayName("+")
+    @ParameterizedTest
+    @CsvSource("1,2")
+    void  plusTest(double a, double b) {
+        assertThat(3.0).isEqualTo(operator.plus(a,b));
```

Comment:
assertJ API 용법에 맞게, assertThat(테스트 값).isEqualTo(예상 값) 으로 하셔야 합니다:)
```suggestion
        assertThat(operator.plus(a,b)).isEqualTo(3.0);
```